src/pages/palanbyulChoi/Main/Main.js

```diff
@@ -1,8 +1,127 @@
 import React, { Component } from 'react';
-
+import Nav from '../../../components/Nav/Nav';
+import Article from './Article';
+import StoryFeed from './StoryFeed';
+import StoryData from './StotyData';
+import RecommendFeed from './RecommendFeed';
+import RecommendData from './RecommendData';
+import './Main.scss';
 class MainPalanbyul extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      feedLoveColor: 'black',
+      feedLoveClassName: 'far fa-heart',
+      commentLoveClassName: 'far fa-heart',
+      feedList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/feedData.json', {
```

Comment:
- `http://localhost:3000` 은 생략해 주세요! 포트 번호가 달라지면(ex. `localhost:3001`) fetch 함수를 일일이 찾아서 코드를 수정을 해주어야 하기 때문에 작업이 복잡해 질 수 있습니다. 
```suggestion
    fetch('/data/feedData.json', {
```

Reply:
감사합니다 👍👍